src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -268,25 +268,30 @@ class EmojiPicker extends React.Component<IProps, IState> {
             } else {
                 emojis = cat.id === "recent" ? this.recentlyUsed : DATA_BY_CATEGORY[cat.id];
             }
-            emojis = emojis.filter((emoji) => this.emojiMatchesFilter(emoji, lcFilter));
-            emojis = emojis.sort((a, b) => {
-                const indexA = a.shortcodes[0].indexOf(lcFilter);
-                const indexB = b.shortcodes[0].indexOf(lcFilter);
-
-                // Prioritize emojis containing the filter in its shortcode
-                if (indexA == -1 || indexB == -1) {
-                    return indexB - indexA;
-                }
-
-                // If both emojis start with the filter
-                // put the shorter emoji first
-                if (indexA == 0 && indexB == 0) {
-                    return a.shortcodes[0].length - b.shortcodes[0].length;
-                }
-
-                // Prioritize emojis starting with the filter
-                return indexA - indexB;
-            });
+
+            if (lcFilter) {
```

Comment:
implicit cast to boolean here

Reply:
this might actually be a case where I agree with the coding style, since I now have to remember which strings are implicitly truthy.